test.js

```diff
@@ -221,6 +221,9 @@ test('data URL', t => {
 	// Lowercase the MIME type.
 	t.is(normalizeUrl('data:TEXT/plain,foo'), 'data:text/plain,foo');
 
+	// Strip empty hash.
+	t.is(normalizeUrl('data:,foo# '), 'data:,foo');
```

Comment:
Are you sure it should do that? Isn't the `#` also part of the actual data?

Reply:
Strings after `#` is hash, at least chrome treat url like this. open `data:,foo%23` and `data:,foo#` to see